sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/version.hpp

```diff
@@ -14,7 +14,7 @@
 #define AZURE_STORAGE_BLOBS_VERSION_MAJOR 12
 #define AZURE_STORAGE_BLOBS_VERSION_MINOR 0
 #define AZURE_STORAGE_BLOBS_VERSION_PATCH 0
-#define AZURE_STORAGE_BLOBS_VERSION_PRERELEASE "beta.4"
+#define AZURE_STORAGE_BLOBS_VERSION_PRERELEASE "beta.5"
```

Comment:
Reminder to make sure to also update the changelog files, and review/add all the necessary details to them, when you are ready.

Reply:
done in separate PR https://github.com/Azure/azure-sdk-for-cpp/pull/974